spec/models/upload_concepts.rb

```diff
@@ -98,4 +105,19 @@
             end
         end
     end
+    
+    describe "destroying concepts" do
```

Comment:
I'll change it to destroying tags. I don't really agree with putting this in t2c rspec because the changes are made in the concept model.